src/microbe_stage/ISpawner.cs

```diff
@@ -29,7 +30,9 @@ public abstract class ISpawner
     /// <value><c>true</c> if destroy queued; otherwise, <c>false</c>.</value>
     public bool DestroyQueued { get; set; }
 
-    public abstract List<ISpawned> Spawn(Node worldNode, Vector3 location);
+    public Stack<Action> QueuedSpawns { get; set; }
+
+    public abstract int Spawn(Node worldNode, Vector3 location);
```

Comment:
Basically I'd want this to be `public abstract ISpawnAction Spawn(Node worldNode, Vector3 location);`
which would return a ISpawnAction in case this wants to spawn entities (for compound clouds this would return null).

Reply:
Where ISpawnAction would be a new class that would have a single method `Node SpawnNext()` that would get called by the spawn system to spawn the next thing. Returning null would indicate everything is spawned.
Then the spawn system would store the ISpawnAction if it didn't have enough room to spawn everything in it.
Before running the spawn logic the spawn system would check if it has an uncompleted ISpawnAction, and if it does it would execute the next step of that before looking for a new spawner to start spawning from.